mods/essb/moves.js

```diff
@@ -1409,9 +1409,8 @@ exports.BattleMovedex = {
 			this.attrLastMove('[still]');
 			this.add('-anim', source, "Toxic", target);
 		},
-		isViable: true,
```

Comment:
This line isn't necessarily needed, I completely agree, but if we do make a data command isViable would mark it a certain way.